cms/djangoapps/contentstore/config/waffle.py

```diff
@@ -83,3 +83,19 @@ def waffle_flags():
     flag_name='library_authoring_mfe',
     module_name=__name__,
 )
+
+
+# .. toggle_name: studio.pages_and_resources_mfe
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to link existing studio views to the new Pages and Resources experience.
+# .. toggle_use_cases: temporary, open_edx
+# .. toggle_creation_date: 2021-03-11
+# .. toggle_target_removal_date: None
```

Comment:
@xitij2000 as we discussed its a temporary flag, would you mind adding an expected removal date? 
thanks.

Reply:
I've tentatively added the removal date for the end of this year. That gives us one release to roll out the MFE. Do tell if that is OK.

Reply:
sounds ok to me. I am ok with this PR we should merge this.